tools/deploy_ui.py

```diff
@@ -31,8 +31,8 @@ def main():
 
     if tag == "latest":
         log.warning("No hash specified. Will run the deployment generation script from the top of master branch")
-    else:
-        cmd += f" && git reset --hard {tag}"
+    #else:
```

Comment:
Although this works in your case, wouldn't this break the deployment with tag = `v2.29.0` for example?